torch/testing/_internal/common_methods_invocations.py

```diff
@@ -236,8 +236,9 @@ def __init__(self,
                                             # function around gradcheck (testing._internal.common_utils.gradcheck)
                  inplace_variant=_NOTHING,  # explicitly pass the inplace variant of the operator if required
                  method_variant=_NOTHING,  # explicitly pass the method variant of the operator if required
-                 has_fake_function=False  # does this opinfo use a function to simulate functional behavior
+                 has_fake_function=False,  # does this opinfo use a function to simulate functional behavior
                                             # of a mutating op
+                 assert_jit_shape_analysis = False,  # assert that jit shape analysis fully propagates shape
```

Comment:
Will people writing OpInfos know to set this properly?

Reply:
i'm not really expecting people to add shape analysis at least initially, it's still very early in implementation and has a few rough edges

Reply:
OK, as the long as the default is false and it won't yell at people